src/lib/Updates.svelte

```diff
@@ -44,20 +60,20 @@
                 </div>
             </div>
         </div>
-        <div class="self-stretch h-[216px] flex-col justify-start items-start gap-1 flex">
-            <div class="self-stretch h-[216px] pb-2 flex-col justify-start items-start gap-[30px] flex">
+        <div class="flex flex-col items-start self-stretch justify-start gap-1">
+            <div class="self-stretch pb-2 flex-col justify-start items-start gap-[30px] flex">
                 <div class="self-stretch text-[#282828] text-base font-normal font-['Inter'] leading-[29px]">{@html truncatedContent}</div>
```

Comment:
also for the `truncatedContent()` function, we could use tailwindcss to do that for use using the `truncate` and `line-clamp-3` _(3 for amount of lines to show)_